src/thunderbots/.bazelrc

```diff
@@ -0,0 +1,19 @@
+# Use C++ 17
+build --cxxopt='-std=c++17'
+
+# Use Our Remote Cache To Avoid Rebuilding Files Already Built By CI
+build --remote_http_cache=https://storage.googleapis.com/thunderbots_bazel_cache_us_multi_region
+
+# Disable sandboxing, since using the remote cache with it enabled is still 
+# considered "experimental"
+--spawn_strategy=standalone
+
+# Don't Try to Upload Results (since CI does not currently have required credentials)
+build --remote_upload_local_results=false 
+
+# Credentials for Uploading Build Results To Our Remote Cache
+#build --google_credentials="/home/gareth/Downloads/tbots-bazel-cache-key.json"
```

Comment:
These credentials are commented out? And these also aren't generic credentials

Reply:
Leaving this unresolved so I remember to come back to it.

Basically I just have this line so I can update the online cache until we get it working in CI.

Reply:
Made an issue to actually use the credentials in CI (#829) and linked it in this file.